content/docs/hooks-reference.md

```diff
@@ -6,11 +6,11 @@ prev: hooks-custom.html
 next: hooks-faq.html
 ---
 
-*Hooks* are an upcoming feature that lets you use state and other React features without writing a class. They're currently in React v16.8.0-alpha.1.
+*Hooks* son una característica que próximamente permitirá usar el estado y otras características de React sin escribir una clase. Están actualmente en React v16.8.0-alpha.1.
 
-This page describes the APIs for the built-in Hooks in React.
+Esta página describe las API para los Hooks incorporados en React.
 
-If you're new to Hooks, you might want to check out [the overview](/docs/hooks-overview.html) first. You may also find useful information in the [frequently asked questions](/docs/hooks-faq.html) section.
+Si eres nuevo en Hooks, es posible que desees revisar [la descripción general](/docs/hooks-overview.html) primero. También puede encontrar información útil en la sección [preguntas frecuentes](/docs/hooks-faq.html).
```

Comment:
I am going to quote @gaearon here referring to translations of his personal blog. I think it is a very concise way to express what a good translation is:

> Regarding quality. Meaning is important. Meaning shouldn't be distorted. However translations shouldn't feel mechanical. If you have a choice between following the original word-for-word or expressing the same meaning in a much idiomatic way, go for the idiom. English is somewhat noun-friendly but conglomerations of nouns can be unreadable in some other languages. My suggestion is that you read a sentence, think how you'd express the same thought in yours, and then write down the sentence you got.

I think this is a better way to convey the same meaning without feeling mechanical.

```suggestion
Si los Hooks son nuevos para ti, es posible que desees revisar primero [la descripción general](/docs/hooks-overview.html). También puedes encontrar información útil en la sección de [preguntas frecuentes](/docs/hooks-faq.html).
```